src/components/ui/autocomplete.tsx

```diff
@@ -214,12 +212,17 @@ export default function Autocomplete({
           variant="outline"
           role="combobox"
           aria-expanded={open}
-          className={cn("w-full justify-between", className)}
+          className={cn("min-w-0 justify-between", className)}
```

Comment:
can we not change this and instead pass a className to override `w-full` if necessary? 🤔 

this would break many other existing usages